docs/basics/intro_information.ipynb

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
<p>For step one, it reads as though you mean "click here to access any dandiset from the allen institute". Maybe say "access an example 2-Photon Dandiset from the allen institute openscope project"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>